projects/core/components/textfield/textfield-multi/textfield-multi.style.less

```diff
@@ -0,0 +1,150 @@
+@import '@taiga-ui/core/styles/taiga-ui-local.less';
+@import '@taiga-ui/core/styles/components/textfield.less';
+
+tui-textfield[multi] {
+    overflow: hidden;
+    align-items: stretch;
+    cursor: text;
+
+    &::before,
+    &::after {
+        block-size: var(--t-height);
+    }
+
+    .t-scroll {
+        .fullsize(absolute, inset);
+
+        top: 0.25rem;
+        bottom: 0.25rem;
+        min-block-size: 0;
+    }
+
+    .t-container {
+        .scrollbar-hidden();
+
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        overflow: scroll;
+        scroll-behavior: smooth;
```

Comment:
Should we use `var(--tui-scroll-behavior)` ?

https://github.com/taiga-family/taiga-ui/blob/3c13004394fca47f6429addcfb215d6e7a28ed6d/projects/core/components/root/root.component.ts#L56

Reply:
The same question for `scroll-behavior` below